packages/tsconfig/client.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "Client",
+  "compilerOptions": {
+    "lib": ["ESNext"],
+    "target": "ESNext",
```

Comment:
I wonder if we should pick a specific target here since `esnext` is a moving target. Maybe `es2022`?